.circleci/config.yml

```diff
@@ -507,12 +507,6 @@ workflows:
 
   fork-check:
     jobs:
-      - bazel_stylelint:
-          <<: *filter_forks
-      - bazel_unit_test:
-          <<: *filter_forks
-      - bazel_build:
-          <<: *filter_forks
```

Comment:
We cannot remove these targets on forked pull requests. this would mean that we don't run tests and stylelint on forks

Reply:
We can't currently - even if the check above works the bazel script fails afterwards. So in the meantime i'd suggest removing them entirely.

Reply:
https://app.circleci.com/pipelines/github/dynatrace-oss/barista/8420/workflows/e751e198-fe99-4ecd-aa61-2962371d7635/jobs/50407 see for the error

Reply:
Thx to @furtnerl I had some pointers where to go next - have another look, i think this is more down the road that you had in mind.